single-degree.php

```diff
@@ -60,6 +60,25 @@
 				</div>
 			<?php endif; ?>
 			</div>
+			<?php echo main_site_degree_display_subplans( $post->ID ); ?>
+			<?php if ( isset( $post_meta['degree_online_url'] ) && ! empty( $post_meta['degree_online_url'] ) ) : ?>
+			<aside class="online-callout">
+				<a class="media-background-container d-block h-100 py-3 px-3 px-lg-4 hover-parent bg-inverse hover-text-black text-decoration-none" href="<?php echo $post_meta['degree_online_url']; ?>" target="_blank">
+					<img src="<?php echo $online_bg_img; ?>" alt="" class=" media-background object-fit-cover hover-child hover-child-scale-up" data-object-fit="cover">
```

Comment:
Looking at it in dev again after a while, what do you think about removing the `hover-child-scale-up` class? I'm thinking that just the color change might be enough for that callout.